apps/web/abi/OtomItemsCore.json

```
@@ -1,842 +1,823 @@
-        { "internalType": "string[7]", "name": "_defaultTierImageUris", "type": "string[7]" },
-        {
-          "components": [
-            { "internalType": "enum ComponentType", "name": "componentType", "type": "uint8" },
-            { "internalType": "uint256", "name": "itemIdOrOtomTokenId", "type": "uint256" },
-            { "internalType": "uint256", "name": "amount", "type": "uint256" },
-            {
-              "components": [
-                { "internalType": "enum PropertyType", "name": "propertyType", "type": "uint8" },
-                { "internalType": "uint256", "name": "minValue", "type": "uint256" },
-                { "internalType": "uint256", "name": "maxValue", "type": "uint256" },
-                { "internalType": "bool", "name": "boolValue", "type": "bool" },
-                { "internalType": "bool", "name": "checkBoolValue", "type": "bool" },
-                { "internalType": "string", "name": "stringValue", "type": "string" },
-                { "internalType": "bool", "name": "checkStringValue", "type": "bool" },
-                { "internalType": "bytes32", "name": "bytes32Value", "type": "bytes32" },
-                { "internalType": "bool", "name": "checkBytes32Value", "type": "bool" }
-              ],
-              "internalType": "struct PropertyCriterion[]",
-              "name": "criteria",
-              "type": "tuple[]"
-            }
-          ],
-          "internalType": "struct BlueprintComponent[]",
-          "name": "_blueprint",
-          "type": "tuple[]"
-        },
-        {
-          "components": [
-            { "internalType": "string", "name": "typeName", "type": "string" },
-            { "internalType": "string", "name": "valueString", "type": "string" },
-            { "internalType": "uint256", "name": "valueNumber", "type": "uint256" },
-            { "internalType": "enum TraitType", "name": "traitType", "type": "uint8" }
-          ],
-          "internalType": "struct Trait[]",
-          "name": "_traits",
-          "type": "tuple[]"
-        },
-        { "internalType": "address", "name": "_mutatorContract", "type": "address" },
-        { "internalType": "uint256", "name": "_ethCostInWei", "type": "uint256" },
-        { "internalType": "address", "name": "_feeRecipient", "type": "address" }
-      ],
-      "name": "updateNonFungibleItem",
-      "outputs": [],
-      "stateMutability": "nonpayable",
-      "type": "function"
-    },
-    {
-      "inputs": [
-        { "internalType": "uint256", "name": "_tokenId", "type": "uint256" },
-        { "internalType": "address", "name": "_owner", "type": "address" },
-        { "internalType": "bytes", "name": "_data", "type": "bytes" }
-      ],
-      "name": "useItem",
-      "outputs": [],
-      "stateMutability": "nonpayable",
-      "type": "function"
-    }
-  ]
-}
+[
+  { "inputs": [], "stateMutability": "nonpayable", "type": "constructor" },
```

need to keep json standard format and keep abi key to be valid
```suggestion
{
  "abi": [
    { "inputs": [], "stateMutability": "nonpayable", "type": "constructor" },
```